File: Front/ProEventos-App/src/app/models/Evento.ts
import { Lote } from './Lote';
import { RedeSocial } from './RedeSocial';
import { Palestrante } from "./Palestrante";

export interface Evento {
  id: Number;
  imagemUrl?: string;
  tema: string;
  local: string;
  data: Date;
  qtdPessoas: number;
  lotes: Lote[];
  redesSociais: RedeSocial[];
  eventosPalestrantes: Palestrante[];
}

File: Back/src/ProEventos.Domain/Models/Evento.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEventos.Domain.Models
{
    public class Evento
    {
        public int Id { get; set; }
        public string? ImagemUrl { get; set; }
        public string Tema { get; set; }
        public string Local { get; set; }
        public DateTime Data { get; set; }
        public int QtdPessoas { get; set; }
        public IEnumerable<Lote> Lotes { get; set; }
        public IEnumerable<RedeSocial>? RedesSociais { get; set; }
        public IEnumerable<EventoPalestrante> EventosPalestrantes { get; set; }
    }
}
